### DISCUSSION TITLE
XCH transfer issue

### DISCUSSION THREAD

**Discussion by sekeli** [Chia-Network/chia-blockchain#18083] [Support]
Hello, I initiated an XCH delayed transfer on May 20th without setting up automatic confirmation. After the time, I did not manually confirm. The XCH sent so far has not arrived at the specified address and has not been returned to the original wallet. May I ask how to solve this problem?Looking forward to your reply. Thank you!

**Comment by sekeli** [comment]
Is this a new version bug? The current status is that both the sender and receiver's wallets display confirmation in progress, there are no actionable buttons, and unconfirmed transactions cannot be deleted. How should I solve it?

**Comment by sekeli** [comment]
Block ID:0xf44a7f3c3e6ac5975bc13e0bc1b632a693157dadaaa09403883c754effb0733d

**Comment by BrandtH22** [comment]
Hey @sekeli , it looks like we have started working with you in discord to resolve the above. 
That discord discussion will be the most timely and thorough way to go through this process but to reiterate the instructions have been provided below:

once the clawback time ends it allows the recipient to claim the funds but the sender can still claim them. If this transaction is between two of your own wallets there should be no issues 👍 You can use the force flag if the clawback is stuck pending:

Get the clawback transaction id by running `chia wallet get_transactions --clawback`

Force the claim by running:
`chia rpc wallet spend_clawback_coins '{"coin_ids":["TRANSACTION_ID"], "force":true, "fee":0}'`

Note - If on windows you will need to escape the quotes for the clawback claim command:
`chia rpc wallet spend_clawback_coins '{\"coin_ids\":[\"TRANSACTION_ID\"], \"force\":true, \"fee\":0}'`

Note if you are not getting a transaction id for the clawback from the first command then I would resync the wallet and run it again (stop chia, rename the wallet DB, restart chia and let it sync)

**Reply by sekeli** [reply]
> Hey @sekeli , it looks like we have started working with you in discord to resolve the above. That discord discussion will be the most timely and thorough way to go through this process but to reiterate the instructions have been provided below:
> 
> once the clawback time ends it allows the recipient to claim the funds but the sender can still claim them. If this transaction is between two of your own wallets there should be no issues 👍 You can use the force flag if the clawback is stuck pending:
> 
> Get the clawback transaction id by running `chia wallet get_transactions --clawback`
> 
> Force the claim by running: `chia rpc wallet spend_clawback_coins '{"coin_ids":["TRANSACTION_ID"], "force":true, "fee":0}'`
> 
> Note - If on windows you will need to escape the quotes for the clawback claim command: `chia rpc wallet spend_clawback_coins '{\"coin_ids\":[\"TRANSACTION_ID\"], \"force\":true, \"fee\":0}'`
> 
> Note if you are not getting a transaction id for the clawback from the first command then I would resync the wallet and run it again (stop chia, rename the wallet DB, restart chia and let it sync)

Thank you, the issue has been resolved.

**Comment by BrandtH22** [comment]
Thank you for the confirmation and let us know if any other questions arise, happy farming! 🌱
